### DISCUSSION TITLE
How are you people handling this: parked car on the street and motion triggered by plants/trees makes clips with zone specified

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#6587] [Ask A Question]
Scenario:
- a camera facing the driveway and road
- two zones created with one for driveway only and other for road only.
- added masks for any known plants/trees/sky
- set automation to notify when vehicle enters zone-driveway or zone-road

But when the automation is on, and a car parked on the street, i get automation triggers every min saying "vehicle found in driveway camera", i am watching the clips all the time, all don't see any other vehicles passing or any people, all i can say is tree/sun shadow/far away birds.  But the notification is "a vehicle found"

Please let me know how you handle this situations.

**Comment by NickM-27** [comment]
You need to make your automation only notify when you want a notification

**Reply by balajeek** [reply]
yes my automation is created using the blueprint and i specifiy the camera and the zone.  But my point is the vehicle is parked for the whole day and i don't need notification every min saying 'vehicle found' when there is not (except the parked car), if another vehicle pass by and i get notified then it makes sense.

What i remember while ago the developer saying they inspect everytime when there is some motion and then try detects objects like vehicle and so, in theory if a plant swaing over wind would trigger motion and in result frigate tells there is a vehicle.  I know its a tricky, but i wonder i wyze or other motion sensor camera does it and i don't get notification like this.

**Reply by NickM-27** [reply]
This is the expected behavior for the events topic, stationary objects are updated once a minute. Like I said, you need to adjust your blueprint to ignore stationary objects.

**Reply by balajeek** [reply]
you mean just add more masks ?

**Reply by NickM-27** [reply]
Not at all, I mean you just need to change the settings in the blueprint to not notify on stationary objects. I'm not sure which blueprint you're using but recent blueprints don't have this issue.

**Comment by balajeek** [comment]
I am on the latest beta:
Is this what you mean ?
![image](https://github.com/blakeblackshear/frigate/assets/19213088/6c85462b-06e4-4fe7-a2f7-3fa07861bf81)

**Reply by NickM-27** [reply]
What do you mean by latest beta? There are many different blueprints and SgtBattens is the recommended blueprint.

**Reply by balajeek** [reply]
yes thats what is shows: 
![image](https://github.com/blakeblackshear/frigate/assets/19213088/a7385c6a-ee3e-4760-a571-fcc89450d108)

**Reply by NickM-27** [reply]
I'd suggest asking for support there then, but as far as I know this behavior should be configurable.

**Reply by balajeek** [reply]
okay will do, thanks.
